index.html

```diff
@@ -602,6 +602,577 @@ <h5 class="red-logo-h5"><strong>START YOUR DAY WITH OUR TOP FIVE ARTICLES</stron
             </div>
           </div>
         </div>
+        <!-- ======================MILESTONE THREE========================= -->
+        <section class="in-the-magazine">
+          <div class="d-flex justify-content-center my-4">
+            <div class="d-flex align-items-center">
+              <span class="heavy fs-24 icon font-roboto bg-orange">N</span>
+              <h4 class="fs-19 heavy font-roboto m-0">IN THE MAGAZINE</h4>
+            </div>
+          </div>
+
+          <section class="row border-top pt-4">
+            <div class="col-12 col-md-3">
+              <div class="top-img-div">
+                <a href="#">
+                  <img class="card-img" src="/img/newsweekpage.jpg" alt="">
```

Comment:
```suggestion
                  <img class="card-img" src="./img/newsweekpage.jpg" alt="">
```
Without dot your image is not visible.

Reply:
okay, corrected that